### PR TITLE
Add Scopescan.ai to the explorer doc section

### PR DESCRIPTION
**Pull request by RedHorse823**
**What changed? Why?**

I add Scopescan, a data analytic platform (similar to Arkham & Nansen) to the block explorer documentation page.

This is because since we have Arkham on this page, Scopescan also support to check Base transactions & tokens & addresses, they also have other nice features like:

1. moneyflow tool to visualize token flows
2. Token holder graph to visualize token holder distributions
3. Smartmoney dashboard to check who made the most money on Base

**Notes to reviewers**

Scopescan is power by 0xscope, they received legit fundings from OKX and Hashkey.

I believe Scopescan can help a lot with Base users with their data and tooling ability. That's why i propose this change. Hope it get approved, thanks!

Scopescan.ai is their platform link, you can try and decide.

### CODE REVIEW

**Comment by wbnns** [comment]
@RedHorse823 

Thanks for taking the time to open this PR! We've added this to our backlog, pending to be reviewed.

**Comment by RedHorse823** [comment]
Hi Team is this repo okay to be merged? Just asking, thanks! @cb-heimdall

**Review by wbnns — CHANGES_REQUESTED** [review]
@RedHorse823 

Hi there! Thanks for this contribution! Can you please update this PR to list Scopescan alphanumerically in order with the others in the list?

File: apps/base-docs/docs/tools/block-explorers.md [hunk 1]
```diff
@@ -30,6 +30,16 @@ Arkham is a crypto intelligence platform that systematically analyzes blockchain
 
 ---
 
+## Scopescan
+
+Scopescan supports [Platform](https://scopescan.ai/home?network=base) Base on their platform.
```
Comment (wbnns) [inline apps/base-docs/docs/tools/block-explorers.md:35]: Please update link to new domain

File: apps/base-docs/docs/tools/block-explorers.md [hunk 2]
```diff
@@ -30,6 +30,16 @@ Arkham is a crypto intelligence platform that systematically analyzes blockchain
...
+
+Base voyagers can check Base transaction records, addresses, tokens, and even smart moneys on their platforms.
```
Comment (wbnns) [inline apps/base-docs/docs/tools/block-explorers.md:37]: s/and even smart moneys/and other traits/

s/platforms/platform/

File: apps/base-docs/docs/tools/block-explorers.md [hunk 3]
```diff
@@ -30,6 +30,16 @@ Arkham is a crypto intelligence platform that systematically analyzes blockchain
...
 ---
 
+## Scopescan
+
+Scopescan supports [Platform](https://scopescan.ai/home?network=base) Base on their platform.
+
+Base voyagers can check Base transaction records, addresses, tokens, and even smart moneys on their platforms. 
+Scopescan also supports visualizing capital flows and checking token holder graphs on Base.
+
+Scopescan is an on-chain data analytics platform that greatly enhances the on-chain analytical experience."
```
Comment (wbnns) [inline apps/base-docs/docs/tools/block-explorers.md:40]: s/on-chain/onchain/

Please remove `"` at the end of the sentence.

**Comment by wbnns** [comment]
@RedHorse823 

Heya, since it has been some time and we haven't heard back, I'm going to go ahead and close this PR for now. If you'd like to pursue these changes in the future please let us know and we can reopen. 👍

**Pull request closed without merging**